core/runtime/src/main/java/org/apache/isis/core/runtime/persistence/adapter/PojoAdapter.java

```diff
@@ -123,20 +118,12 @@ public void replacePojo(final Object pojo) {
         this.pojo = pojo;
     }
 
-
-    // -- getOid, replaceOid
+    // -- getOid
     @Override
     public Oid getOid() {
         return oid;
     }
 
-    @Override
-    public void replaceOid(Oid persistedOid) {
```

Comment:
field in oid in ObjectAdapter is now final
oid replacement no longer allowed

Reply:
+1  This looks like it's the main significant change in this PR... with the replaceOid stuff now removed.